CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ The format is based on [Keep a Changelog].
 
 [#59]: https://github.com/raxod502/ctrlf/issues/59
 
+* Customize keybindings by modifying the mode map `ctrlf-mode-map`
+  instead of the `ctrlf-mode-bindings` variable. `ctrlf-mode-bindings`
+  is obsolete but remains available.
+
+[#64] https://github.com/raxod502/ctrlf/pull/64
+
```

Comment:
Would you mind putting the bullet points next to each other, and the link anchors together at the bottom?